DEVELOPERS.md

```diff
@@ -148,10 +153,26 @@ To debug code, and to see changes in real time, it is often useful to have a loc
     http://localhost:8082/
     ```
 
-For more information on Supabase Studio, see the [Supabase Studio readme](./studio/README.md).
+## What's Next?
+
+Make your changes to the file(s) you'd like to update.
+
+### Open a pull request
+
+When you're done making changes and you'd like to propose them for review by opening a pull request.
+
+### Submit your PR & get it reviewed
+
+- Once you submit your PR, others from the Unleash community will review it with you. The first thing you're going to want to do is a self review.
+- After that, we may have questions, check back on your PR to keep up with the conversation.
+- Did you have an issue, like a merge conflict? Check out GitHub's git tutorial on how to resolve merge conflicts and other issues.
+
+### Your PR is merged!
+
+Congratulations! The whole Supabase community thanks you 🙏🏼
 
-## Build in Public
+Once your PR is merged, you will be proudly listed as a contributor in the [contributor chart](https://github.com/supabase/supabase/graphs/contributors).
```

Comment:
New change, let's not tell people how to open a pull request and issue. They can find that information elsewhere.

Reply:
I suppose this information should be given especially to beginners (referred few organizations and their repos)

Reply:
There is plenty of info available online on how to open a PR. Keep the doc short and succinct with only crucial info included. we shouldn't be assuming that readers are beginners.

Also, Unleash❓